transformation/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would it be possible to "transform" this into a `*.properties` files like the rest of our Log4J configuration?  Having a lone, oddball XML file is not ideal.